source/speechViewer.py

```diff
@@ -27,6 +36,31 @@ def onClose(self, evt):
 			return
 		evt.Veto()
 
+	def onDestroy(self, evt):
+		log.debug("SpeechViewer Destroyed")
+		self.savePositionInformation()
+		evt.Skip()
+
+	def doDisplaysMatchConfig(self):
+		configSizes = config.conf["speechView"]["displays"]
+		attachedSizes = self.getAttachedDisplaySizes()
+		convertedAttachedSizes = [repr( (i.width, i.height) ) for i in attachedSizes]
+		return len(configSizes) == len(attachedSizes) and all( configSizes[i] == convertedAttachedSizes[i] for i in range(0, len(configSizes)))
+
+	def getAttachedDisplaySizes(self):
+		displays = (wx.Display(i) for i in range(wx.Display.GetCount()))
+		return [display.GetGeometry().GetSize() for display in displays]
+
+	def savePositionInformation(self):
+		position = self.GetPosition()
+		config.conf["speechView"]["x"] = position.x
+		config.conf["speechView"]["y"] = position.y
+		size = self.GetSize()
+		config.conf["speechView"]["width"] = size.width
+		config.conf["speechView"]["height"] = size.height
+		config.conf["speechView"]["displays"] = self.getAttachedDisplaySizes()
+		config.conf["speechView"]["autoPositionWindow"] = False

```

Comment:
As discussed with @michaelDCurran add these to the config schema. Perhaps the displays entry could just hold the hash of the object. Though now I'm thinking about this again, using the hash would have the disadvantage that it becomes much harder to verify the config / behaviour.